src/modules/__tests__/popup.test.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { getCommentCount } from '../popups.js';
+import mockPopup from '../__mocks__/popup.mock.js';
+
+test('Add x comments and check count', () => {
+  const x = 10;
+  mockPopup(x);
+  const test = getCommentCount();
```

Comment:
- I see that you testing the `getCommentCount` function the previous reviewer mentioned in the last review in this file, instead of the nav.test.js, that's alright. great work 👍🏼